src/screens/FeedScreen/hooks/useFeedPreloadHook.ts

```diff
@@ -0,0 +1,28 @@
+import {NativeScrollEvent, NativeSyntheticEvent} from 'react-native';
+
+import useFeedUtilsHook from './useFeedUtilsHook';
+import {FEEDS_DIFF_TO_FETCH} from '../../../utils/constants';
+
+const useFeedPreloadHook = (feedLength: number, fetchCallback: () => void) => {
+  const {getCurrentPostViewed} = useFeedUtilsHook();
+
+  function __shouldFetchNextFeeds(momentumEvent: NativeSyntheticEvent<NativeScrollEvent>): boolean {
+    const shownIndex = getCurrentPostViewed(momentumEvent);
+    console.log(`shown index ${shownIndex} vs ${feedLength}`);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Logging sensitive information such as the index of the currently viewed post can pose a security risk. Consider removing this log or ensure that it's only enabled in development environments.
```diff
- console.log(`shown index ${shownIndex} vs ${feedLength}`);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->